README.md

```diff
@@ -85,25 +83,57 @@ Counter.count;  // 3, own property
 SubCounter.count;  // 2, own property
 ```
 
-See [Initializing fields on subclasses](#initializing-fields-on-subclasses) for more details on an alternative proposed semantics and why it was not selected.
+### Why not to reinitialize public fields on subclasses
 
-## Static private methods and accessors
+Kevin Gibbons has proposed that class fields have their initialisers re-run on subclasses. This would address the static private subclassing issue by adding those to subclasses as well, leading to no TypeError on use.
 
-### Semantics
+With this alternate, the initial counter example would have the following semantics:
+
+```js
+// NOTE: COUTNERFACTUAL SEMANTICS BELOW
+static Counter {
+  static count = 0;
+  static inc() { this.count++; }
+}
+class SubCounter extends Counter { }
+
+Counter.hasOwnProperty("count");  // true
+SubCounter.hasOwnProperty("count");  // true
+
+Counter.count; // 0, own property
+SubCounter.count; // 0, own property
+
+Counter.inc();  // undefined
+Counter.count;  // 1, own property
+SubCounter.count;  // 0, own property
+
+// ++ is just dealing iwth own properties the whole time
```

Comment:
Typo: iwth -> with

Reply:
Thanks for this correction. There are lots of typos here; probably better for others not to spend time looking for them until I do another pass over things.